client/src/pages/proposalDetails/ProposalDetails.js

```diff
@@ -0,0 +1,58 @@
+import './ProposalDetails.css';
+import ProgressBar from 'react-bootstrap/ProgressBar'
+
+var IS_ADMIN = true;
+var DEADLINE = '3/5/21';
+var AMOUNT = 14360;
+var PERCENT_VOTED = 60;
+var PERCENT_YES = 20;
+var DUMMY_DESCRIPTION = `The Earth Action Initiative will organize a variety of
+events and actions that build community, leverage existing
+efforts, inspire new plans, and find innovative ways to
+execute these actions through effective communication.`
+
+/* Takes in a number and converts it to a dollar amount string w/ commas
+placed appropriately (every 3 spaces); does not include dollar sign
+*/
```

Comment:
for documenting, use 
/**
*/
function

(https://gomakethings.com/whats-the-best-way-to-document-javascript/)